packages/backend/src/api/resources/OrganizationMembership.ts

```diff
@@ -18,7 +18,7 @@ export class OrganizationMembership {
     return new OrganizationMembership(
       data.id,
       data.role,
-      data.public_metadata,
+      data.public_metadata || {},
```

Comment:
Do we need this default value? My assumption is that the API always returns `{}`

Reply:
There's an ongoing discussion on slack about this. I went through the backend code and I think that we always default to `{}` even if all struct use `json.RawMessage` which translates to `null | {}`

I will revert this one for now, however we need to keep in mind that we cannot simply add `{}` as a fallback in clerk_go, as in theory, people could be checking like `!metadata` which will return `false` for `{}`